libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.scss

```diff
@@ -73,6 +73,16 @@ ion-header ion-toolbar {
   --border-width: 0;
   --background: transparent;
   --color: #{get-color('black')};
+
+  ion-title {
+    $spacingToTop: 30px;
```

Comment:
would be nice to use a defined size here - but I tried it out and came to the same conclusion as you did here

Reply:
I wanted to use the sizes, but after some trying, I figured, as you did, a locally scoped variable would be best. This speaks into a greater issue with the modal header (and how it is connected to page), as I've discussed a few times with @jakobe. I'm afraid my solution won't work well with supporting #1171, so we should decide whether to go with my solution and accept that the solution for #1400 has to be conditional or address the bigger issue with the modal header.

Reply:
For the record, in relation to the modal header, I consider this to be atleast a strike 2 (strike 1 was #1217). I'm unaware of previous strikes, and can't find any related issues or PRs, which could be considered 1 more strike.